.travis.yml

```diff
@@ -12,6 +12,10 @@ matrix:
     env:
     - OS_NAME: macos
     osx_image: xcode10.2
+    cache:
+    directories:
+      - $HOME/Library/Caches/Homebrew
+      - /usr/local/Homebrew
```

Comment:
ah, well done, this is a trick that we can use also in other places. (is it intended to have `cache:` and `directories:` at the same level of nesting?)

Reply:
Ah you are right, we need to nest it one extra level.

I also didn't fully understand if what I am doing here is going to be entirely effective. This SO answer https://stackoverflow.com/questions/39930171/cache-brew-builds-with-travis-ci makes this thing look much more complicated than I think it should be.

Looking at that above post I concluded these are most likely the main paths we want to cache.